### ISSUE TITLE
only work on browser?

### ISSUE THREAD

**Issue by bluelovers** [ibitcy/react-stores#35]
> ReferenceError: window is not defined

```ts
import { useStore, Store } from 'react-stores';
import { ICachedJSONRowPlus } from 'build-json-cache/lib/types';

export interface IDataListStoreState
{
	dataList: ICachedJSONRowPlus[]
}

export const dataListStore = new Store<IDataListStoreState>({
	dataList: [],
});

export default dataListStore

console.log(dataListStore.state.dataList);

```

https://github.com/ibitcy/react-stores/search?q=window&unscoped_q=window

**Comment by konstantin24121** [comment]
@bluelovers no, react-stores should works in any es-environment. We will fix this bug asap

**Comment by ruslanchek** [comment]
@bluelovers try now, fixed in 5.0.3
